src/main/java/mods/railcraft/common/blocks/aesthetics/glass/BlockStrengthGlass.java

```diff
@@ -90,8 +90,15 @@ public void initializeDefinition() {
     }
 
     @Override
-    public void finalizeDefinition() {
-        IRailcraftBlock.super.finalizeDefinition();
```

Comment:
Removing this breaks the color handler.

Reply:
This is not removed; as the implementation for this method is removed, `finalizeDefinition` now inherits the default implementation from `IRailcraftBlock`.